generated/mocks/mocks/buffer.go

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 // Automatically generated by MockGen. DO NOT EDIT!
-// Source: /Users/xichen/gocode/src/github.com/m3db/m3db/storage/buffer.go
+// Source: /Users/r/go/src/github.com/m3db/m3db/storage/buffer.go
```

Comment:
welp..maybe I should modify the mockgen process to remove the GOPATH part of the path so we don't end up changing the mock files when someone different regenerates them.